### DISCUSSION TITLE
[Feature Request] Vim Base Keymap

### DISCUSSION THREAD

**Discussion by ghost** [zed-industries/zed#15497] [Feature Requests]
Hello, I would first like to say that I know I can replace everything on my own, but it would be great to see Zed developers provide an opinionated set of keymaps for Vim. Just for clarification, I’m not referring to Vim mode, but Vim keymaps throughout the Zed editor (Workspaces, Panes, Docks, Terminals, etc.).

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
